ckanext/stadtzhtheme/templates/base.html

```diff
@@ -10,9 +10,10 @@
 
 {% block styles %}
 {{ super() }}
-  {% resource 'stadtzhtheme/jquery-ui.css' %}
+  {% resource 'stadtzhtheme/auto-complete.css' %}
   {% resource 'stadtzhtheme/stadtzhtheme.css' %}
   {% resource 'stadtzhtheme/masonry.pkgd.js' %}
-  {% resource 'stadtzhtheme/jquery-ui.js' %}
-  {% resource 'stadtzhtheme/ogdzh_autocomplete.js' %}
+  {% resource 'stadtzhtheme/auto-complete.js' %}
+  {% resource 'stadtzhtheme/autocomplete-ogdzh_search.js' %}
+  {% resource 'stadtzhtheme/autocomplete-sitewide-search.js' %}
```

Comment:
I think naming both custom-resources with `ogdzh` makes the naming a little more consistent.
What about naming one `autocomplete-ogdzh-homepage-search` and the other `autocomplete-ogdzh-header-search`?